test_ssgs.py

```
@@ -364,7 +364,7 @@ def compute_path_spectral_cost(ssgs, path):
     print("✓ Cost function verified: cost = -log(L) + smoothness_weight * smoothness + 0.25 * psycho_weight")
     
     print("\n✓ TEST PASSED: A* decoder correctly implements smoothness-aware decoding")
-    return ssgs, astar_path, viterbi_path
+    return None
```

Ruff (0.15.18)

[warning] Do not explicitly `return None` in function if it is the only possible return value

Remove explicit `return None`

(RET501)